templates/careers/engineering.html

```diff
@@ -6,82 +6,216 @@
     </div>
   </div>
 </section>
 
-{% include "shared/_social-icons-section-with-github.html" %}
+<section class="p-section u-no-padding--top">
+  <hr class="is-fixed-width p-rule">
+  <div class="u-fixed-width">
+    <h2>See for yourself the work we're doing</h2>
+  </div>
+  <div class="row">
+    <div class="col-6 col-medium-3 col-start-large-7 col-start-medium-4">
+      <ul class="p-inline-list is-inlined-with-h2">
+        <li class="p-inline-list__item">
+            <a class="p-icon--github" href="https://github.com/canonical" data-size="large" aria-label="Follow @canonical on GitHub"></a>
+        </li>
+        <li class="p-inline-list__item">
+            <a href="https://www.linkedin.com/company/234280" class="p-icon--linkedin">LinkedIn</a>
+        </li>
+        <li class="p-inline-list__item">
+            <a href="https://twitter.com/Canonical" class="p-icon--twitter">Twitter</a>
+        </li>
+        <li class="p-inline-list__item">
+            <a href="https://www.youtube.com/user/celebrateubuntu" class="p-icon--youtube">YouTube</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
 
 
 {% with section_classes='p-strip', heading_topic='Engineering', tag_id='1564', limit='3' %}
-  {% include "shared/_blog-card.html" %}
+<noscript>
+  <section class="p-section u-hide {% if section_classes %}{{ section_classes }}{% else %}p-strip{% endif %}" data-js="latest-news">
+    <div class="row">
+      <div class="row">
+        <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default p-rule">Blogs</h2>
+        <h3 class="p-heading--2">Find out more about {{ heading_topic }}</h3>
+      </div>
+    </div>
+  </section>
+</noscript>
+
+<section class="u-no-padding--top p-section u-hide {% if section_classes %}{{ section_classes }}{% else %}p-section{% endif %}" data-js="latest-news">
+  <hr class="is-fixed-width p-rule">
+  <div class="row">
+    <div class="col-6 col-medium-3">
+      <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Blogs</h2>
+      <h5 class="p-heading--2">Find out more about <br class="u-hide--small"/> {{ heading_topic }}</h5>
+    </div>
+    <div id="latest-articles" class="col-6 col-medium-3" style="padding-top: 0.5rem">
+      <div style="min-height: 9.1rem">
+        <i class="p-icon--spinner u-animation--spin">Loading...</i>
+      </div>
+    </div>
+  </div>
+  <template style="display:none" id="articles-template">
+    <article class="u-extra-space">
+      <div class="blog-p-card__content">
+        <hr class="p-rule">
+        <h5 class="p-rule">
+          <a class="article-title"></a>
```

Comment:
There is no href's here.

Reply:
This was there before, isn't this going to be added by the `CanonicalLatestNews` library?

Reply:
Added empty href